### PR TITLE
Rename manager TEST to TEST-Ion

### PR DESCRIPTION
**Pull request by SchoolGuy**
Rename the TEST env to TEST-Ion as it is exclusively for the Ion Squad.

### CODE REVIEW

File: terracumber_config/tf_files/SUSEManager-Test-Ion-NUE.tf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Mandatory variables for terracumber
 variable "URL_PREFIX" {
   type = string
-  default = "https://ci.suse.de/view/Manager/view/Manager-Test/job/manager-TEST-acceptance-tests"
+  default = "https://ci.suse.de/view/Manager/view/Manager-Test/job/manager-TEST-Ion"
```
Comment (meaksh) [inline terracumber_config/tf_files/SUSEManager-Test-Ion-NUE.tf:4]: This should be:

```suggestion
  default = "https://ci.suse.de/view/Manager/view/Manager-Test/job/manager-TEST-Ion-acceptance-tests"
```

File: terracumber_config/tf_files/SUSEManager-Test-Ion-NUE.tf [hunk 2]
```diff
@@ -105,7 +105,7 @@ module "cucumber_testsuite" {
   images = ["rocky8o", "opensuse154o", "sles15sp4o", "ubuntu2204o"]
 
   use_avahi    = false
-  name_prefix  = "suma-test-"
+  name_prefix  = "suma-testion-"
```
Comment (meaksh) [inline terracumber_config/tf_files/SUSEManager-Test-Ion-NUE.tf:108]:
```suggestion
  name_prefix  = "suma-test-ion-"
```

File: terracumber_config/tf_files/SUSEManager-Test-Ion-NUE.tf [hunk 3]
```diff
@@ -129,11 +129,17 @@ module "cucumber_testsuite" {
         mac = "aa:b2:93:01:00:41"
         memory = 12288
       }
+      additional_repos = {
+        server_stack = "http://download.suse.de/ibs/Devel:/Galaxy:/Manager:/TEST/images/repo/SLE-Module-SUSE-Manager-Server-4.3-POOL-x86_64-Media1/"
+      }
```
Comment (meaksh) [inline terracumber_config/tf_files/SUSEManager-Test-Ion-NUE.tf:134]: Is there a reason to add this? We don't really need this IIUC.

```suggestion
```

File: terracumber_config/tf_files/SUSEManager-Test-Ion-NUE.tf [hunk 4]
```diff
@@ -129,11 +129,17 @@ module "cucumber_testsuite" {
...
     }
     proxy = {
       provider_settings = {
         mac = "aa:b2:93:01:00:42"
       }
+      additional_repos = {
+//        proxy_stack = "http://download.suse.de/ibs/SUSE:/Maintenance:/17859/SUSE_Updates_SLE-Module-SUSE-Manager-Proxy_4.1_x86_64/",
+      }
```
Comment (meaksh) [inline terracumber_config/tf_files/SUSEManager-Test-Ion-NUE.tf:142]:
```suggestion
```

File: terracumber_config/tf_files/SUSEManager-Test-Ion-NUE.tf [hunk 5]
```diff
@@ -194,6 +200,9 @@ module "cucumber_testsuite" {
         memory = 2048
         vcpu = 2
       }
+      additional_repos = {
+//        Test_repo = "http://download.suse.de/ibs/Devel:/Galaxy:/Manager:/salt-testing:/sle15sp2/standard/"
+      }
```
Comment (meaksh) [inline terracumber_config/tf_files/SUSEManager-Test-Ion-NUE.tf:205]:
```suggestion
```

File: terracumber_config/tf_files/SUSEManager-Test-Ion-NUE.tf [hunk 6]
```diff
@@ -208,6 +217,9 @@ module "cucumber_testsuite" {
       provider_settings = {
         mac = "aa:b2:93:01:00:4e"
       }
+      additional_repos = {
+        Test_repo = "http://download.suse.de/ibs/Devel:/Galaxy:/Manager:/salt-testing:/sle15sp3/standard/"
+      }
```
Comment (meaksh) [inline terracumber_config/tf_files/SUSEManager-Test-Ion-NUE.tf:222]:
```suggestion
```

**Review by meaksh — CHANGES_REQUESTED** [review]

**Review by meaksh — CHANGES_REQUESTED** [review]
I just noticed we are also missing to change the messages for the generated email subject at: https://github.com/SUSE/susemanager-ci/blob/9ae948245c042e6e665574e6ffc46cb56b0bd551/terracumber_config/tf_files/SUSEManager-Test-Ion-NUE.tf#L30 and https://github.com/SUSE/susemanager-ci/blob/9ae948245c042e6e665574e6ffc46cb56b0bd551/terracumber_config/tf_files/SUSEManager-Test-Ion-NUE.tf#L40

**Comment by meaksh** [comment]
@SchoolGuy as for the other test environments, we want to use a custom `Results TEST-ION` instead of generic `Result TEST`, to identify the environment in the email subject we will receive with the results.

Like:

https://github.com/SUSE/susemanager-ci/blob/f94c453c098f359fd0e74a60baa5f99214722deb/terracumber_config/tf_files/SUSEManager-Test-Orion-NUE.tf#L30

**Comment by SchoolGuy** [comment]
@meaksh Fixed.

**Review by meaksh — APPROVED** [review]
LGTM

**Pull request merged**